Makefile

```diff
@@ -15,7 +15,7 @@ include Makefile.docker
 
 setup: ## Setup local dev env.
 setup:
-		virtualenv -p python3 .env && . .env/bin/activate && \
+		virtualenv -p python3 .local_env && . .local_env/bin/activate && \
 	  pip install -r requirements/development.txt
```

Comment:
It simply didn't work when I wanted to use `make setup` because the target dir `.env` for the virtualenv was colliding with the `.env` custom settings django file. Clear sign no one has used the `make setup` in this repo before if you asked me. 
I added the docs covering local env and local test runs. (in Readme.md)

Reply:
I see in the cookiecutter we call this directory `.venv`, so changing here accordingly